src/shared/schema/avl.schema.ts

```diff
@@ -55,22 +59,22 @@ export const vehicleActivitySchema = z.object({
     ValidUntilTime: z.string().min(1),
     VehicleMonitoringRef: z.coerce.string().nullish(),
     MonitoredVehicleJourney: z.object({
-        LineRef: z.coerce.string().nullish(),
+        LineRef: z.coerce.string().regex(NM_TOKEN_REGEX).nullish(),
         DirectionRef: z.union([
```

Comment:
I think the following fields could be made to be NMTOKENS:
- DirectionRef
- VehicleMonitoringRef
- ItemIdentifier